tests/unit/amo/components/TestCategory.js

```diff
@@ -307,6 +312,7 @@ describe(__filename, () => {
       category,
       errorHandlerId: errorHandler.id,
     }));
+    sinon.assert.calledWith(fakeDispatch, setViewContext(addonType));
```

Comment:
And again, this test was meant to cover [a specific if branch](https://github.com/bobsilverberg/addons-frontend/blob/d41b5e99ba24219eab50f1a1127f937497195fa6/src/amo/components/Category/index.js#L112) so it's not necessary to add an assertion for the view context logic.